bitcoin_transfer/spend_your_coin.md

```diff
@@ -1,8 +1,8 @@
-## Spend your coin {#spend-your-coin}
+﻿## Spend your coin {#spend-your-coin}
 
 So now that you know what a **bitcoin address**, a **ScriptPubKey**, a **private key** and a **miner** are, you will make your first **transaction** by hand.
 
-As you proceed through this lesson you will add code line by line as it is presented to build a method that will leave feedback for the book in a Twitter style message.
+As you proceed through this lesson you will add code line by line as it is presented to build a method that will leave feedback for the book in a Twitter style message. I _highly recommend_ you to follow the instructions first on the TestNet and after do them again on the Main Bitcoin network.
```

Comment:
I agree, but all the samples in the chapter were and are on TestNet, so to avoid confusion I wanted to make clear that it is better to start with the testnet and then go on and repeat them on mainnet.

Reply:
On my side I am OK with switching to testnet. The first chapter is still using Mainnet which is enough to "feel" your money on the line. Also now, with the high fees, such experience everywhere could be very costly. :)

Reply:
Ah wait! This is the first chapter.